src/math/trig.c

```diff
@@ -3,574 +3,100 @@
 // Copyright(c) 2016 Intel Corporation. All rights reserved.
 //
 // Author: Seppo Ingalsuo <seppo.ingalsuo@linux.intel.com>
-//         Liam Girdwood <liam.r.girdwood@linux.intel.com>
-//         Keyon Jie <yang.jie@linux.intel.com>
-
+// Liam Girdwood <liam.r.girdwood@linux.intel.com>
+// Keyon Jie <yang.jie@linux.intel.com>
```

Comment:
not sure why this is a diff but regardless this fixes the indentation

Reply:
Actually before the authors were all aligned. Shouldn't we have an "Author: " thingy on each of the authors with SPDX actually?

Reply:
Oh I see what is going on, I missed the Author: prefix when I edited the whitespace. Thanks for pointing that out.